KsApi/models/graphql/ManagePledgeViewBackingEnvelopeTests.swift

```diff
@@ -605,7 +696,20 @@ final class ManagePledgeViewBackingEnvelopeTests: XCTestCase {
             "imageUrl": "http://www.kickstarter.com/avatar.jpg",
             "id": "VXNlci0xMjA3OTk3NjQ5",
             "uid": "1100793144",
-            "name": "Creator McBaggins"
+            "name": "Creator McBaggins",
+            "storedCards": [
```

Comment:
It might be worth it (later) to store these as separate `templateFragments` (just strings) that we can compose these json files with in the future. They are quite big.

Reply:
Justin left a few `FIXME` in his original few migrations that mention the need to add templates for these models and updating the tests. We should start doing that.